substrate/substrate-test-runtime-client/src/lib.rs

```diff
@@ -260,24 +260,27 @@ impl<B> TestClientBuilderExt<B>
 	fn build_with_longest_chain(
 		self,
 	) -> (Client<B>, sc_consensus::LongestChain<B, substrate_test_runtime::Block>) {
-		self.build_with_native_executor(None)
+		self.build_with_native_executor(Some(new_native_executor()))
```

Comment:
Do we really want native though? I'd prefer to use compiled WASM instead, especially with native executor going away in the future.

Reply:
Well, It's actually `NativeElseWasmExecutor` https://github.com/subspace/subspace/pull/1210/files#diff-49e30e984a0da673301f6d55c117ceb3124cc6d072cfdc35a09d76edf22e16b6L278 which is still the standard right now.